index.html

```diff
@@ -1,9 +1,73 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <title>Lesson 5</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Personal Portfolio of Susana Sanchez" />
+    <meta name="keywords" content="Portfolio, About Me, Experience, Contact" />
+    <meta name="author" content="Susana Sanchez" />
+    <title>Lesson 7</title>
+    <style>
+      body {
+        font-family: Arial, sans-serif;
+        margin: 20px;
+        line-height: 1.6;
+      }
+      h1,
+      h2 {
+        color: #333;
+      }
+      ul {
+        list-style-type: square;
+      }
+      img {
+        border-radius: 50%;
+        box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+      }
+    </style>
   </head>
   <body>
-    <h1>Hello from Lesson 5!</h1>
+    <h1>Susana Sanchez Lopez</h1>
+
+    <section id="About">
+      <h2>About</h2>
+      <img
+        src="https://pbs.twimg.com/media/FOjXSZ1XIAMQXWJ?format=jpg&name=large"
+        alt="Picture of Susana Sanchez"
+        width="200"
+        height="200"
+        style="float: left; margin-right: 10px"
+      />
+      <p>
+        I was born and raised in Colombia. I work as a Relationship Banker at
+        Bank of America, and in my spare time, I'm a coding enthusiast.
+      </p>
+    </section>
+
+    <section id="Experience">
+      <h2>Experience</h2>
+      <ul>
+        <li>Python Coding Course</li>
+        <li>Computer Science Pathway Completion</li>
+        <li>Student of the Year 2022 in Gwinnett County - CTEA Pathways</li>
```

Comment:
I didn't count this off since its a minor issue, but you only have two of the three links in the connect section.

Not a big deal :) 

If you were struggling to find a coding-relevant link for your connect section, you could add a link in another section too.
It might be cool to include a link to CTEA Pathways here or an article about you being student of the year in 2022 - thats a cool achievement!